### ISSUE TITLE
Documentation / limits on product configurations?

### ISSUE THREAD

**Issue by sam327** [facebookincubator/facebook-for-woocommerce#138]
I've got products that are showing up in the business manager catalog, but not showing up in the page's "Shop" tab. And I don't really know where to begin debugging w/ the plugin. 

Is there (or could there be) any documentation on the limits of what we can put in? For example, we've got products that have a spot to add a custom engraving - and it doesn't look like FB likes that. Or, if your product is missing X, Y, or Z, it won't get added to your page. 

Because it looks like I've got everything talking to everything else - but something is failing along the way. So, if there were some nice guidelines about common things to fix, I could at least exhaust that list before posting newbie issues here.

**Comment by tahoff** [comment]
Can you give me a bit more information about your current setup? Specifically, which version are you using (you can update to 1.6.5 if you're not using it?)? What is your shop url? Is there anything specific about the products which aren't appearing (are they hidden, out of stock, or have some other commonality)?

**Comment by dmitridr** [comment]
Sounds related to #134. If you give us a page ID we can check if you were blacklisted by the shops team.

**Comment by sam327** [comment]
Can I email one of you store info? I'd rather not put client out here on github... 

I just converted the site's FB page to the business manager, a couple days before posting this. I've installed 1.6.5 of the FB plugin. (Note: If you click through after creating a pixel, the download that FB gives you is 1.6.0)

I have zero products in my shop page - and when a user tries to go to the shop page, we get an error. 

<img width="570" alt="screen shot 2017-11-09 at 10 15 55 am" src="https://user-images.githubusercontent.com/1746360/32612744-0ccbf664-c537-11e7-89b7-4350cc4a63d0.png">

Regarding stock... On most items, we don't actually track stock, as they're custom made-to-order things. But, in the catalog in FB, I see everything as saying "in stock."
